testsuite/cluster-test/src/cluster_swarm/mod.rs

```diff
@@ -10,6 +10,17 @@ use futures::{future::try_join_all, try_join};
 
 #[async_trait]
 pub trait ClusterSwarm {
+    async fn remove_all_network_effects(&self) -> Result<()>;
+
+    /// Runs the given command in a container against the given Instance
+    async fn run(
+        &self,
+        instance: Instance,
```

Comment:
Instance could be reference too to avoid clone?

Reply:
Done